snuba/query/snql/parser.py

```diff
@@ -1198,7 +1198,7 @@ def validate_entities_with_query(
                 entity = get_entity(node.data_source.key)
                 try:
                     for v in entity.get_validators():
-                        v.validate(query)
+                        v.validate(query, alias)
```

Comment:
Previously, no validators made use of the `alias` parameter. Now we need to use it for the Column validator in case of composite queries.